### PR TITLE
Basic command line options enhancement

### PR DESCRIPTION
**Pull request by vadzimt**
1. Build result added to .gitignore
2. Option names changed
3. Added `--port` option

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,7 +33,8 @@ CTOP is a tool which allows you to quickly find out what's happening on a machin
 * Metrics are for one node only (not cluster-wide)
 
 ## Notes
-* CTOP should run on anything, but I've only tested it on Ubuntu.
+* CTOP should run on anything. Originally was developed on Ubuntu, runs
+  great on CentOS and MacOS X.
```
Comment (crufter) [inline README.md]: Can we change this to: "Originally was developed on Ubuntu, and it's reported to run on CentOS and MacOS X"?

Great is relative 8)

**Comment by crufter** [comment]
I am not familiar MX4J so I don't know how correct is the option name change. Otherwise it looks good to me :+1:

**Comment by chrusty** [comment]
The suggested changes look fine to me, but I'd like to keep the port called "MX4JPort" rather than "cassandraPort", because this port is opened by the MX4J plugin rather than by cassandra itself.

This should remind people that they need to be running Cassandra with MX4J if they find that it doesn't work.

The rest of the stuff looks good though.

**Comment by vadzimt** [comment]
@chrusty thanks!

**Pull request merged**